MIP106/MIP106.md

```diff
@@ -577,11 +508,11 @@ Anti-cheat formula:
 
 #### 6.4.1: Special SPK Token Pre-farming
 
-The Support Facilitators can activate a new SPK token pre-farming airdrop program to incentivize adoption of overcollateralized perpetual yield exposure in new markets.
+The Support Facilitators can activate a new SPK token pre-farming airdrop program to capture other growth opportunities.
 
 The program can last until the moment SPK launches, or a shorter duration, which must be declared by the Support Facilitators. The exact details of the special pre-farming airdrop program is specified in *6.4.1.1A*, which the Support Facilitators can modify at will, as long as it conforms the specifications of *6.4.1*.
```

Comment:
- "Is" should be changed to "are" when referring to "details"
- I would propose adding "section" before "6.4.1.1A" - this will make clear that this is what the Support Facilitators can modify at will, and not the "details" that the beginning of the sentence mark as the subject. If we leave it as is, I would propose changing "as long as it conforms" to "as long as they conform".